spec/index.bs

```diff
@@ -11392,6 +11614,19 @@ attachments used by this encoder.
                         - |this|.{{GPURenderPassEncoder/[[occlusion_query_active]]}} is `true`.
                     </div>
                 1. Set |this|.{{GPURenderPassEncoder/[[occlusion_query_active]]}} to `false`.
+
+                1. [$Enqueue a render command$] on |this| which issues the subsequent steps on the
+                    [=Queue timeline=] with |renderState| when executed.
+            </div>
+            <div data-timeline=queue>
+                [=Queue timeline=] steps:
+
+                1. Let |passingFragments| be the number of fragment samples that have passed all
+                    the per-fragment tests since the corresponding
+                    {{GPURenderPassEncoder/beginOcclusionQuery()}} command was executed.
```

Comment:
I'm not sure it's guaranteed to actually be the number of fragment samples. I don't understand occlusion queries very well but I seem to remember the results can vary?

Tangentially this comment says we have both "precise" and "non-precise" occlusion queries but afaict we only have one type. And I don't know what the difference is. https://github.com/gpuweb/gpuweb/issues/72#issuecomment-911588426

Reply:
The "Precise" vs. "non-precise" mentioned probably corresponds with [this setting from Metal](https://developer.apple.com/documentation/metal/mtlvisibilityresultmode?language=objc), [this flag in Vulkan](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#queries-occlusion), and [the OCCLUSION/BOOLEAN_OCCLUSION flags in D3D12](https://learn.microsoft.com/en-us/windows/win32/api/d3d12/ne-d3d12-d3d12_query_type), all of which allow you to ask specify that you either want a specific fragment count or a simple boolean. I'm assuming the boolean is a fair bit faster to generate.

Looking back through the code commits adding occlusion queries I found this comment in which there was going to be a `preciseOcclusion` boolean in the `beginOcclusionQuery` call, but you asked for it to be removed. It wasn't based on any technical reason, though. You simply weren't sure *how* the value should be passed (optional param or dictionary arg). So... maybe we need to revisit this to ensure we're not unintentionally dropping a valuable feature on the floor?

In any case, it seems obvious from the linked PR comment that the imprecise variant was to be considered the default, so I've changed the text here to only specify writing a non-zero value if any fragments pass.

Reply:
Yup, Kai re-surfaced that and we captured it in a feature request at #3715